acvm-repo/acir/codegen/acir.cpp

```diff
@@ -915,18 +915,7 @@ namespace Circuit {
             static ToLeRadix bincodeDeserialize(std::vector<uint8_t>);
         };
 
-        struct PermutationSort {
-            std::vector<std::vector<Circuit::Expression>> inputs;
-            uint32_t tuple;
-            std::vector<Circuit::Witness> bits;
-            std::vector<uint32_t> sort_by;
-
-            friend bool operator==(const PermutationSort&, const PermutationSort&);
-            std::vector<uint8_t> bincodeSerialize() const;
-            static PermutationSort bincodeDeserialize(std::vector<uint8_t>);
-        };
-
-        std::variant<ToLeRadix, PermutationSort> value;
+        std::variant<ToLeRadix> value;
```

Comment:
If this variant can only be a `ToLeRadix` now, shouldn't we use that directly instead of a variant?

We should probably also add a comment that if a new directive is added it should be written as a variant of each directive again.

Reply:
This code is auto-generated, we could modify the Directives so that it does not put a variant, but we have already an issue to remove the directives and this will be resolved by the issue.